components/raftstore/src/store/peer.rs

```diff
@@ -2146,23 +2144,12 @@ impl Peer {
             return false;
         }
 
-        // Should we call pre_propose here?
-        let last_pending_read_count = self.raft_group.raft.pending_read_count();
-        let last_ready_read_count = self.raft_group.raft.ready_read_count();
-
         poll_ctx.raft_metrics.propose.read_index += 1;
-
         self.bcast_wake_up_time = None;
-        let id = Uuid::new_v4();
-        self.raft_group.read_index(id.as_bytes().to_vec());
-
-        let pending_read_count = self.raft_group.raft.pending_read_count();
-        let ready_read_count = self.raft_group.raft.ready_read_count();
 
-        if pending_read_count == last_pending_read_count
-            && ready_read_count == last_ready_read_count
-            && self.is_leader()
-        {
+        // Should we call pre_propose here?
```

Comment:
Could you elaborate on your consideration?

Reply:
Actually, it was not my comment, and I think we can remove it.

Reply:
@5kbpers What about your comments?

Reply:
This comment is originally from old code (see #2149)

Reply:
The comment had removed.